motif-docs/pages/guides/datadog/support-for-fargate.mdx

```diff
@@ -31,10 +31,20 @@ Here is a complete example:
     "datadog": {
       "enabled": true,
       "datadogSite": "us5.datadoghq.com", //plaintext
-      "datadogApiKey": "DD_API_KEY.from.parameter.store" //from parameter store or secrets manager
+      "datadogApiKey": "DD_API_KEY.from.parameter.store", //from parameter store or secrets manager
+      "envVariables":{
+        // Environment variables to pass to the datadog agent
+				// Similar to regular services variables
+				"DD_APM_ENABLED": true,
+				"ANOTHER_VARIABLE":{
+					"fromParameterStore": "parameter.store.name"
+				},
+				"THIRD_VARIABLE":{
+					"fromSecretsManager": "secrets.manager.arn"
+				}
     }
```

Comment:
indentation is messed up 
<img width="601" alt="image" src="https://user-images.githubusercontent.com/8813276/206322543-4de266d3-e402-46d3-81e6-adb34d9d3e12.png">

Reply:
@flybayer should be fixed now.